int8 control_mode
bool is_emergency
